browser/ui/views/tabs/brave_tab_style_views.inc.cc

```diff
@@ -264,8 +274,17 @@ void BraveVerticalTabStyle::PaintTab(gfx::Canvas* canvas) const {
       color_id = kColorBraveVerticalTabSeparator;
     }
 
-    PaintBackgroundStroke(canvas, TabStyle::TabSelectionState::kActive,
-                          widget->GetColorProvider()->GetColor(color_id));
+    SkPath stroke_path =
```

Comment:
Note: the previous implementation was painting a stroke for active tabs (whether pinned or not). Instead of relying on `PaintBackgroundStroke`, this change paints the stroke here.